libs/bilge/src/Bilge/IO.hs

```diff
@@ -82,9 +82,12 @@ newtype HttpT m a = HttpT
 class MonadHttp m where
     getManager :: m Manager
 
-instance Monad m => MonadHttp (HttpT m) where
+instance {-# OVERLAPPING #-} Monad m => MonadHttp (HttpT m) where
     getManager = HttpT ask
 
+instance {-# OVERLAPPABLE #-} (MonadTrans t, MonadHttp m, Monad m) => MonadHttp (t m) where
```

Comment:
This simply allows any transformer over an MonadHttp to derive MonadHttp.

Reply:
1. The monad constraint is necessary because it's a superclass of MonadHttp; are you suggesting I should remove it there?

2. The MonadTrans instance effectively 'recurses' until it hits `HttpT` which is the 'base' instance; I can't remove the base case 😄

Reply:
> 1. The monad constraint is necessary because it's a superclass of MonadHttp; are you suggesting I should remove it there?

no, just my habit of asking stupid questions.  please don't mess with MonadHttp in this PR.

> 2. The MonadTrans instance effectively 'recurses' until it hits `HttpT` which is the 'base' instance; I can't remove the base case smile

ah, right.  thanks!